build.gradle

```diff
@@ -1,5 +1,5 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
-apply from :'buildsystem/dependencies.gradle'
```

Comment:
why you move it to root project ? i think we need to wrap it on the package for custom gradle

Reply:
moved to `buildSrc`